env.example

```diff
@@ -0,0 +1,24 @@
+# this is the externally available public path
+# that connects the client dev server to the correct
+# host, in case you have to override it
+BASE_PATH=/
+API_BASE_PATH=/
+STATIC_DIRECTORY=/static
+PUBLIC=https://twodotoh.a.influxcloud.dev.local/
+UI_SHA=local_dev_mode
+CLOUD_URL=/auth
+
+# NOTE: self signed certs aren't currently supported with
```

Comment:
there's no ssl in this image, so these need to be cleaned up

Reply:
cool 🔥